aztk/spark/cli/cluster_create.py

```diff
@@ -51,7 +49,6 @@ def execute(args: typing.NamedTuple):
         wait=args.wait,
         username=args.username,
         password=args.password,
-        ssh_key=args.ssh_key,
```

Comment:
This was not supported as it was there were just leftover pieces from before. The ssh_key is in the secrets_config, not the cluster_config now.

`aztk spark cluster create` hasn't supported  the ssh_key param since I did the configuration file work. Can add back if need be, though.